DEV-Simple/views/ViewController.swift

```diff
@@ -389,5 +395,32 @@ extension ViewController: WKScriptMessageHandler {
                 notification.notificationOccurred(.success)
             }
         }
+
+        if message.name == "podcast", let message = message.body as? String {
+            switch message {
+            case "play":
+                avPlayer?.play()
+            case "pause":
+                avPlayer?.pause()
+            case "load":
+                let javascript = "document.getElementById('audio').querySelector('source').src"
```

Comment:
Would it possible to send this information as part of the event, as opposed to the injection?

I'm okay with this if we think it's better.

Reply:
It can be done indeed! I played with sending the URL as part of the event itself, separated by a semicolon and then parsing the message itself. I think it might be useful for events like playback speed (sending the 0.5x/1x/2x speed) and "skip ahead" too. I'll update so it works this way soon